.travis.yml

```diff
@@ -169,6 +179,14 @@ script:
   - python -m pytest -v --durations=10 python/ray/rllib/test/test_optimizers.py
   - python -m pytest -v --durations=10 python/ray/rllib/test/test_evaluators.py
 
+  # streaming tests
+  - python -m pytest -v --durations=10 test/test_batched_queue.py
+  - python -m pytest -v --durations=10 test/test_logical_graph.py
+  
+  # streaming tests
+  - python -m pytest -v --durations=10 test/test_batched_queue.py
+  - python -m pytest -v --durations=10 test/test_logical_graph.py
```

Comment:
These tests seem to appear twice. Is that intentional?

Reply:
I think I fixed that. They appear only once in my branch.